composer/profiler/json_trace.py

```diff
@@ -22,8 +22,8 @@
 
 
 class JSONTraceHandler(ProfilerEventHandler):
-    """Records trace events in `JSON trace format
-    <https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview>`_.
+    """Records trace events in `JSON trace format <https://\\
```

Comment:
needs to be on a seperate line so `make style` doesn't add a period, which would break the link.